internal/querynode/query_collection.go

```diff
@@ -398,6 +422,34 @@ func (q *queryCollection) consumeQuery() {
 	}
 }
 
+func (q *queryCollection) doSearchRequests() {
+	for {
+		select {
```

Comment:
how to exit  normaly ?

Reply:
sorry, my bad, I forget it. Now it has been fixed.